api/v1/eks_cloudwatchlogs_types.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2024 Tigera, Inc. All rights reserved.
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+)
+
+// EksCloudwatchLogsDeployment is the configuration for the EksCloudwatchLogs Deployment.
+type EksCloudwatchLogsDeployment struct {
```

Comment:
Since the deployment name is `eks-log-forwarder`, shouldn't this also be called EKSLogForwarderDeployment?
Either that, or we should rename the deployment to `eks-cloud-watch`. I think the former is probably better.

Reply:
Done